### ISSUE TITLE
undefined method `to_sym' for ["active_storage_analysis", 4]:Array

### ISSUE THREAD

**Issue by javinto** [deanpcmad/sidekiq-limit_fetch#153]
On updating to Sidekiq 7 and sidekiq-limit-fetch I run into this exception:

```
undefined method `to_sym' for ["active_storage_analysis", 4]:Array
queue.process_limit = @process_limits[queue_name.to_s] || @process_limits[queue_name.to_sym]
                                                                                        ^^^^^^^
Did you mean?  to_s
               to_set
/Users/me/.rvm/gems/ruby-3.1.2@project/gems/sidekiq-limit_fetch-4.4.0/lib/sidekiq/limit_fetch/queues.rb:121:in `apply_process_limit_to_queue'
```

In my sidekiq configuration file I have defined:
:queues:
  - [active_storage_analysis, 4]
  - [active_storage_purge, 1]
  - [mailers, 2]
  - default
  - content

It looks like Sidekiq-limit-fetch cannot handle queue priorities anymore.

**Comment by deanpcmad** [comment]
Can you try using the master branch and see if that latest commit fixes it?

**Comment by javinto** [comment]
Same issue with Master branche unfortunately.

**Comment by uichi** [comment]
Hi, everyone 😀 ! I tried to fix this problem in #155. I'm not familiar with `sidekiq-limit_fetch` so I can't guarantee the fixes.
